cmd/note_common.go

```diff
@@ -94,11 +94,9 @@ func noteRunFn(cmd *cobra.Command, args []string) {
 	createNote(rn, isMR, int(idNum), msgs, filename, linebreak, commit, true)
```

Comment:
I appreciate the nice commit message but please don't @ people in commit messages,
because github will send a notification email whenever someone pushes that commit
to a public fork. I haven't checked if they it's possible to turn that off.

The essence of the change is that we pass *both* line numbers for context lines, to please gitlab.

Reply:
Oops, I had no idea.  Fixed.